lib/ello_click/affiliate/link.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElloClick.Affiliate.Link do
+  defstruct [original: nil, affiliated: nil, is_affiliated: false]
```

Comment:
Is it worth expanding out the names of these fields a bit as long as we're trying to be extra-clear (e.g. `original_link`, `affiliated_link`)? Not sure if it's idiomatic, jsut a though.